rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -257,14 +257,33 @@ impl LodeSphericalExpansion {
             }
         } else {
             let p_eff = 3.0 - potential_exponent;
-            let factor = std::f64::consts::PI.powf(1.5) * 2.0_f64.powi(p_eff as i32) / gamma(0.5 * potential_exponent);
+            let factor = std::f64::consts::PI.powf(1.5) * (2.0 * smearing_squared).powf(0.5 * p_eff) / gamma(0.5 * potential_exponent);
 
             for k_vector in k_vectors {
                 let k_norm_squared = k_vector.norm * k_vector.norm;
-                let value = gamma_ui(0.5 * p_eff, 0.5 * k_norm_squared * smearing_squared);
-                fourrier.push(factor * value / k_vector.norm.powi(p_eff as i32));
+                let x = 0.5 * k_norm_squared * smearing_squared;
+
+                // Compute the gamma_ui over a power law using analytical
+                // expressions for a more stable Fourier transform of the
+                // density
```

Comment:
@kvhuguenin what's the exact function being computed here? would be nice to add it to the comment